fili-core/src/test/java/com/yahoo/bard/webservice/web/endpoints/JobsEndpointResources.java

```diff
@@ -49,7 +49,7 @@ public static ApiJobStore getApiJobStore() {
         fieldValueMap1.put(DefaultJobField.JOB_TICKET, "ticket1");
         fieldValueMap1.put(DefaultJobField.DATE_CREATED, "2016-01-01");
         fieldValueMap1.put(DefaultJobField.DATE_UPDATED, "2016-01-01");
-        fieldValueMap1.put(DefaultJobField.QUERY, "https://localhost:9998/v1/data/QUERY");
+        fieldValueMap1.put(DefaultJobField.QUERY, "https://localhost:PORT/v1/data/QUERY");
```

Comment:
Correct me if I'm wrong. I think one way you can do it without using a placeholder port is to put the expected result resource(e.g. `JobsEndpointResources.java`) in the `JobsServletSpec.groovy` so you can initialize your resource after setting up Jersey.